src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,153 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a Ui object.
+ */
+public class Ui {
+    
+    public Ui() {
+    }
+
+    /**
+     * Displays welcome message.
+     */
+    public static void welcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("\n Always at your service, \n" + logo + "\n");
+        System.out.println("Your Majesty, I am your loyal Duke.");
+        System.out.println("I offer a range of administrative services. Do type 'assist' to see the comprehensive list.");
+        System.out.println();
+    }
+
+    /**
+     * Displays a dashed line break that is 32 dashes long.
+     */
+    public static void dashedLineBreak() {
+        String dashedLine = "- ";
+        System.out.println(dashedLine.repeat(32));
+    }
+
+    /**
+     * Displays list of possible user commands.
+     */
+    public static void assist() {
+        dashedLineBreak();
+        System.out.println("Greetings, Your Majesty. \n");
+        System.out.println("Use any of the commands on the left to access my quality services:");
+        System.out.println("\ttodo [TASK]: Adds a todo to your scroll");
+        System.out.println("\tdeadline [TASK] /by [DATE AND/OR TIME]: Adds a deadline to your scroll");
+        System.out.println("\tevent [TASK] /on [DATE AND/OR TIME]: Adds an event to your scroll");
+        System.out.println("\tscroll: Displays your scroll - your list of tasks");
+        System.out.println("\tconquer [NUMBER]: Marks the particular item on your scroll as DONE");
+        System.out.println("\tdelete [NUMBER]: Deletes the particular item from your scroll");
+        System.out.println("\tfind [KEYWORD]: Returns a list of relevant items on your scroll");
+        System.out.println("\tdismiss: This will be my cue to leave.");
+        System.out.println();
+        System.out.println("Now, how may I serve you?");
+        dashedLineBreak();
+        System.out.println();
```

Comment:
Maybe the use of a new line character at the end of every statement can reduce the amount of repeated code?